services/director-v2/src/simcore_service_director_v2/modules/db/repositories/projects_metadata.py

```diff
@@ -0,0 +1,22 @@
+from typing import Any
+
+from models_library.projects import ProjectID
+from simcore_postgres_database.utils_projects_metadata import ProjectMetadata
+from simcore_postgres_database.utils_projects_metadata import (
+    get as projects_metadata_get,
+)
+
+from ._base import BaseRepository
+
+
+class ProjectsMetadataRepository(BaseRepository):
+    async def get_metadata(self, project_id: ProjectID) -> dict[str, Any] | None:
```

Comment:
MINOR: I would specify that is the "custom" metadata. NOTE that there will be more columns in the `project_metadata` table

Reply:
well, if these are not "custom" then they should have a precise name don't you think? in the end everything is a metadata.

Reply:
well, if these are not "custom" then they should have a precise name don't you think? in the end everything is a metadata.